src/openforms/conf/base.py

```diff
@@ -1182,6 +1183,10 @@
     "LOG_OUTGOING_REQUESTS_MAX_AGE", default=7 * 24
 )  # number of hours
 
+# Pdok api locatieserver url
+PDOK_LOCATIE_SERVER_URL = config(
+    "PDOK_LOCATIE_SERVER_URL", "https://api.pdok.nl/bzk/locatieserver/search/v3_1/"
+)
```

Comment:
1. When adding settings, can you please respect the grouping of settings in this file? It now looks like this setting belongs to the django-log-outgong-requests library, which is not the case.

2. I think I'd also rather have this service specified in a run time `zgw_consumers.Service` model (with a default for this URL maybe?) so that it can be changed - many organizations funnel all their API traffic through their gateway, and maybe this also needs to go over the NLX network. We have an existing pattern of configuring external services in the admin, I don't think this warrants deviating from it.